tests/integration/test_remote_convert_nwb_dataset.py

```diff
@@ -1,10 +1,19 @@
+import os
 import pathlib
+import sys
 
 import datalad.api
 import pytest
 
 import nwb2bids
 
+# These tests fail on Windows GitHub CI due to git-annex adjusted branch issues
+_windows_github_ci_xfail = pytest.mark.xfail(
```

Comment:
let's name more consistently with original name, e.g.

```suggestion
pytest_mark_xfail_windows_github_ci = pytest.mark.xfail(
```

and move into a common module among tests for potential reuse in other test files